manifests/autoheal-route.yml

```diff
@@ -19,7 +19,7 @@ kind: Route
 metadata:
   name: autoheal
 spec:
-  host: autoheal.private
+  host: autoheal.example.com
   port:
```

Comment:
I believe this section is not required. It is already stated in the service

Reply:
It's best not to specify the host in the Route object, so OpenShift will automatically assign one based on the service name, namespace, and the cluster's DNS settings.